Dockerfile

```diff
@@ -129,6 +133,5 @@ COPY start /app/bin/
 RUN chmod a+rx /app/bin/start
 
 EXPOSE 8080 4443
-USER 1000
-ENTRYPOINT ["/app/bin/start"]
-CMD ["runserver"]
+USER weblate
```

Comment:
AFAIK the numeric user was needed for OpenStack compatibility, but I'm not sure if that still holds.

Reply:
I wanted to make it more consistent: `adduser` could result non-1000 UID in a different image.
If numeric ID was potentially necessary guess I'd better revert it.

Reply:
It seems still needed, see https://docs.openshift.com/container-platform/4.4/openshift_images/create-images.html